tests/test_models/test_huggingface.py

```diff
@@ -1,13 +1,57 @@
 import logging
 import unittest
+import pytest
+
 
 from brainscore_language.artificial_subject import ArtificialSubject
 from brainscore_language.models.huggingface import HuggingfaceSubject
 
 logging.basicConfig(level=logging.INFO)
 
+
 class TestHuggingfaceSubject(unittest.TestCase):
-    def test_next_word(self):
+
+
+    # @pytest.mark.memory_intense
+    # def test_next_word_mask_bert_base_uncased(self):
+    #     """
+    #     This is a simple test that takes in text = 'the quick brown fox', and asserts
+    #     that the next word predicted is '.'
+    #     This test is a stand-in prototype to check if our model definitions are correct.
+    #     """
+    #     model = HuggingfaceSubject(model_id='bert-base-uncased',
+    #                                 # model_class=AutoModelForCausalLM,
+    #                                 # tokenizer_class=AutoTokenizer,
+    #                                region_layer_mapping={}
+    #                                 )
+    #
+    #     logging.info(' '.join(['Running', model.identifier(), 'for next word prediction test']) )
+    #     text = 'the quick brown fox jumps over the lazy [MASK]'
+    #     model.perform_behavioral_task(
+    #                        task=ArtificialSubject.Task.next_word,
+    #                        )
+    #     next_word = model.digest_text(text)['behavior'].values
+    #     assert next_word == '.'
+
+
+    # def test_next_word_gpt2_xl(self):
+    #     """
+    #     This is a simple test that takes in text = 'the quick brown fox', and asserts
+    #     that the next word predicted is ' jumps'.
+    #     This test is a stand-in prototype to check if our model definitions are correct.
+    #     """
+    #
+    #     model = HuggingfaceSubject(model_id='gpt2-xl',
+    #                                region_layer_mapping={}
+    #                                 )
+    #     text = 'the quick brown fox'
+    #     logging.info(f'Running {model.identifier()} with text "{text}"')
+    #     model.perform_behavioral_task(task=ArtificialSubject.Task.next_word)
+    #     next_word = model.digest_text(text)['behavior'].values
+    #     assert next_word == ' jumps'
```

Comment:
dead code?

Reply:
I was commenting in and out tests as tests. All done, and uncommented.